tests/runtime/out_stackdriver.c

```diff
@@ -1506,5 +1906,14 @@ TEST_LIST = {
     {"default_labels_k8s_resource_type", flb_test_default_labels_k8s_resource_type },
     {"custom_labels_k8s_resource_type", flb_test_custom_labels_k8s_resource_type },
 
+    /* test timestamp */
+    {"Timestamp_format_object_common_case", flb_test_timestamp_format_object_common},
```

Comment:
```suggestion
    {"timestamp_format_object_common_case", flb_test_timestamp_format_object_common},
```

As the field `timestamp` of the log entry is lowercase too, let's make it lowercase. Please apply to all the test names.